pkg/plugins/optional/grafana/v1alpha/scaffolds/edit.go

```diff
@@ -18,14 +18,22 @@ package scaffolds
 
 import (
 	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
 
 	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/optional/grafana/v1alpha/scaffolds/internal/templates"
+
+	"sigs.k8s.io/yaml"
 )
 
 var _ plugins.Scaffolder = &editScaffolder{}
 
+const ConfigFilePath = "grafana/custom-metrics/config.yaml"
```

Comment:
do we use it outside, if not then we do not need the capital prefix.
If we do that we are exporting the const to be used in another places
```suggestion
const configFilePath = "grafana/custom-metrics/config.yaml"
```